Makefile

```diff
@@ -33,6 +33,7 @@ ocaml-deps: .build/local/lib/pkgconfig/libsecp256k1.pc
 		|| opam repository set-url k "$(K_SUBMODULE)/k-distribution/target/release/k/lib/opam"
 	opam update
 	opam switch 4.03.0+k
+	eval $$(opam config env) \
```

Comment:
@msaxena2 why is it ok for ubuntu without this?

Reply:
Probably a different version of opam. This is fine though and I've seen issues in the past when running opam install without first running ``eval `opam config env`, so I'm inclined to say we should accept this change.